geometry_msgs/PoseStamped start_pos
geometry_msgs/PoseStamped end_pos
---
nav_msgs/Path path


